src/game_logic/interfaces.ts

```diff
@@ -0,0 +1,96 @@
+export interface Project {
+  projectName: string
+  description: string
+  tileName: string
+  cost: number
+  revenue: number
+  effort: number
+  happiness: number
+}
```

Comment:
I'd say go ahead and submit- we can keep iterating on this

I think we need to solidify the project process a little. I envision each project as transforming one tile into another - `Clear Debris` transforms `Overgrown Lot` into `Empty Lot`, `Landscape` turns `Empty Lot` into `Park`, and so on

`revenue` and `happiness` are properties of both tiles, not the project, and the player can see the change to those fields (and, crucially, the tags) when choosing which project to pursue.

Projects are simpler:

```TypeScript
export interface Project {
    name: string
    description: string
    toTile: string // tile ID?
    cost: number
    effort: number
}
```

Reply:
let me know if this makes sense; I am also open to other interpretations of projects/contracts, but I would like to make sure I understand whatever one we go with

Reply:
this makes sense, and reduces the number of different entities that we have to understand. I hadn't thought of treating "unimproved" and "improved" tiles as just types of tile. Good idea. Also makes it easier for content creators to just generate a bunch of ideas